test/test_ufbrw.F90

```diff
@@ -84,6 +86,7 @@ program test_ufbrw
 ! open the print file to record ufbrep test results
 
       open(55,file='ufbrw_prnt_out')
+      iprt=2
```

Comment:
Are you even using this new variable any more?  I don't see it being used anywhere later in the code so you can probably just remove it now.